pkg/imgpkg/cmd/pull.go

```diff
@@ -49,6 +58,7 @@ func NewPullCmd(o *PullOptions) *cobra.Command {
 	o.BundleRecursiveFlags.Set(cmd)
 	o.LockInputFlags.Set(cmd)
 	cmd.Flags().StringVarP(&o.OutputPath, "output", "o", "", "Output directory path")
+	cmd.Flags().StringVarP(&o.OutputType, "output-type", "", "", "Will print machine readable output on standard out: [yaml]")
```

Comment:
since we have a flag called --output that takes a directory path, it's pretty confusing to have a --output-type flag that controls unrelated functionality.

Reply:
I thought about that but we already were using output-type for the same thing in describe

Reply:
im not sure tbh. asking others in slack.

Reply:
See also: [#carvel ~ 2022-08-16](https://kubernetes.slack.com/archives/CH8KCCKA5/p1660581869764349?thread_ts=1660581817.485209&cid=CH8KCCKA5)